function handleError(err, result) {
  // console.log(err);
}

module.exports = handleError;
